remotespark/livyclientlib/command.py

```diff
@@ -0,0 +1,50 @@
+import textwrap
+
+from remotespark.utils.guid import ObjectWithGuid
+from remotespark.utils.log import Log
+
+
+class Command(ObjectWithGuid):
+    def __init__(self, code):
+        super(Command, self).__init__()
+        self.code = textwrap.dedent(code)
+        self.logger = Log("Command")
+
+    def __eq__(self, other):
```

Comment:
I use this in the unit tests so that I can easily do assertions -- for example, `thing.assert_called_with(Command('blah'))`